### ISSUE TITLE
CodigoConvenio Não Aparece no Boleto Final

### ISSUE THREAD

**Issue by Leomacedo161** [BoletoNet/BoletoNetCore#270]
Olá, bom dia. Quando crio a conta bancária, um dos valores que eu passo para o objeto é ''1234567'' para a variável CodigoConvenio. Porém, na sequencia do código, depois da criação do boleto e etc. A mesma deveria ser impressa no arquivo final porém o valor que aparece lá é "0000000". Alguém sabe como posso proceder?

**Comment by Leomacedo161** [comment]
Meu chefe diz que o Número do Convênio Líder, que no caso é o CodigoConvenio deveria aparecer naquela linha em destaque. Não a nada que eu possa fazer?

01REMESSA01COBRANCA       00000000000000000000NOME DA EMPRESA  000BANCODOBRASIL  2806230000000                      **0000000**                                                                                                                                                                                                                                                                  000001
00000000003000000082810009000000       101                      000000000000001010000       019000000004DSC1700000053420027062300000000111000010000 12N120623000000000000007400000000000000000000000000000000000000000000000258025342000146NOME DA EMPRESA DOIS                      R RUA 85 COMPLEMENTO FUNDOS 1       BAIRRO     00000000CIDADE      PRSCSYSTEM.COLLECTIONS.GENERIC.LIST`1[EMPR   000002
0000000000623000000011100                                                                                                                                                                                                                                                                                                                                                                                    000003
9                                                                                                                                                                                                                                                                                                                                                                                                         000004

**Comment by Leomacedo161** [comment]
Sim, o layout do banco aponta que naquela região deve conter o o Número do Convênio Líder

**Comment by meirochun** [comment]
@Leomacedo161 Verifica a geração desse arquivo. Este `PRSCSYSTEM.COLLECTIONS.GENERIC.LIST` não deveria estar ai.

**Comment by Leomacedo161** [comment]
> @Leomacedo161 Verifica a geração desse arquivo. Este `PRSCSYSTEM.COLLECTIONS.GENERIC.LIST` não deveria estar ai.

Eu acabei de retirar o responsável por essa mensagem, mas ainda sim os outros valores que eu preciso estão zerados, que no caso são o _Número do Convênio Líder e o Número Sequencia_l. Ambos são exigidos pelo layout do Banco do Brasil.

**Comment by Leomacedo161** [comment]
Olá, alguma atualização sobre, @carloscds? :)

**Comment by carloscds** [comment]
@Leomacedo161 Você ja deu um olhada no código fonte do projeto ?

**Comment by Leomacedo161** [comment]
Então, aparentemente está normal, o valor está sendo passado corretamente e etc. Ele só n está mostrando no arquivo final .txt na posição q o banco do brasil aponta.

**Comment by carloscds** [comment]
@Leomacedo161 Cria um projeto console com todos os dados que você esta passando e manda o link aqui, pode ser no seu github

**Comment by Leomacedo161** [comment]
No código do convenio estou usando um dado interno do nosso banco, que no caso está como 1234567.  E o padrão é o 400

**Comment by carloscds** [comment]
@Leomacedo161 e vc colocou no campo que informei ?

**Comment by Leomacedo161** [comment]
sim, fiz bancoNetCore.Beneficiario.ContaBancaria.CodigoConvenio = bancoNetCore.Beneficiario.Codigo; mas mesmo assim não foi

**Comment by carloscds** [comment]
@Leomacedo161 monta um exemplo no seu Github e manda o link aqui, assim consigo debugar.
